.github/workflows/call-check-advisories.yml

```diff
@@ -0,0 +1,94 @@
+name: Check and publish security advisories
+on:
+  workflow_call:
+    inputs:
+      fetch-key:
+        required: true
+        type: string
+      is-artifact:
+        required: true
+        type: boolean
+      changed-advisories:
+        required: false
+        type: string
+        default: '[]'
+jobs:
+  check-advisories:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: source
+          # We need to retrieve full history to determine the correct
+          # `published` and `modified` timestamps
+          fetch-depth: 0
+      - run: mkdir -p ~/.local/dockerImages
+      - name: Fetch artifact
+        if: ${{ inputs.is-artifact }}
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.fetch-key }}
+          path: ~/.local/dockerImages
+      - name: Fetch cache
+        id: fetch-binaries
+        if: ${{ !inputs.is-artifact }}
+        uses: actions/cache/restore@v3
+        with:
+          key: ${{ inputs.fetch-key }}
+          path: ~/.local/dockerImages
+      - name: Populate cache
+        if: ${{ !inputs.is-artifact && steps.fetch-binaries.outputs.cache-hit != 'true' }}
+        uses: ./.github/workflows/call-nix.yml
+      - name: Fetch cache
```

Comment:
Could you please change the name to "Fetch cache (second attempt after cache miss)"?